packages/core/src/common/message-service-protocol.ts

```diff
@@ -20,13 +20,18 @@ export enum MessageType {
 export interface Message {
     type: MessageType;
     text: string;
+    options?: MessageOptions;
+}
+
+export interface MessageOptions {
```

Comment:
the options were meant to add more hints, so that `MessageClient`s could decide on how to handle them. 

please, move the actions back to `Message`.

Reply:
should be `message.actions`, **not** `message.options!.actions`.